app/scripts/views/sign_up.js

```diff
@@ -226,6 +232,22 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    _checkPasswordStrength: function () {
+      var self = this;
+      var password = this.getElementValue('.password');
+      self.passwordcheck(password, function (passwordCheckStatus) {
+        var err;
+        if (passwordCheckStatus === 'MISSING_PASSWORD') {
+          // same as PASSWORD_REQUIRED
+          err = AuthErrors.toError('PASSWORD_REQUIRED');
+        } else if (AuthErrors.ERRORS[passwordCheckStatus] !== undefined) {
```

Comment:
No need for the `if (...`, if the `passwordCheckStatus` is not in AuthErrors, it will return the string verbatim.